### PR TITLE
EBNT-397 Added some tests

### PR DESCRIPTION
**Pull request by ElmiRakh**

### CODE REVIEW

File: tests/ext/test_docker/test_build_context.py [hunk 1]
```diff
@@ -80,3 +83,15 @@ def _cut_empty_lines(string):
...
+def test_docker_registry_io():
+    registry = DockerIORegistry()
+    client = docker.DockerClient()
+
+    client.images.pull('hello-world:latest')
+
+    assert registry.get_host() == 'https://index.docker.io/v1/'
+    registry.push(client, 'hello-world:latest')
+    image = DockerImage('hello-world')
+    assert registry.image_exists(client, image) is True
```
Comment (mike0sv) [inline tests/ext/test_docker/test_build_context.py:97]: - 'is True'

File: tests/runtime/test_interface.py [hunk 2]
```diff
@@ -50,3 +53,16 @@ def test_interface_descriptor__to_dict(interface: Interface):
...
+def pipeline():
+    return Pipeline('Test Pipeline', [PipelineStep('a', 'b'), PipelineStep('c', 'd')], NumpyNdarrayDatasetType,
+                    NumpyNdarrayDatasetType)
+
+
+def test_pipeline_interface(pipeline):
+    # Dunno if this really helps?
+    iface = pipeline_interface(pipeline)
+    assert 'run' in iface.exposed_methods()
+    assert str(iface.exposed_method_args('run')[0]) == 'data: NumpyNdarrayDatasetType'
```
Comment (mike0sv) [inline tests/runtime/test_interface.py:68]: very strange assert)

Comment (ElmiRakh) [reply tests/runtime/test_interface.py:68]: Well i was trying to check if argument passed correctly from input_data to exposed method "run"

File: tests/ext/test_docker/test_prebuild.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+def test_prebuild__generate_image(prebuild_contextdir):
+    _generate_dockerfile(prebuild_contextdir, '3.2.2')
+    assert 'Dockerfile' in os.listdir(prebuild_contextdir)
+    with open(prebuild_contextdir.join('Dockerfile'), 'r') as fd:
+        assert fd.readlines()[0] == 'FROM python:3.2.2-slim\n'
+
+
+@pytest.mark.skipif(not has_docker(), reason='no docker installed')
```
Comment (mike0sv) [inline tests/ext/test_docker/test_prebuild.py:44]:
```from tests.conftest import docker_test

@docker_test```

Comment (ElmiRakh) [reply tests/ext/test_docker/test_prebuild.py]: fixed

**Pull request merged**